.github/scripts/update_dependency.py

```diff
@@ -32,3 +40,12 @@
         line if package not in line else f'"{package} @ file://{path}/{package}-python",\n'
     )
     print(replaced_line, end="")
+
+for line in fileinput.input("tox.ini", inplace=True):
+    # Ensure that tox uses the working branch for the SDK PR.
+    replaced_line = (
+        line
+        if package not in line
+        else f"    git+https://github.com/amazon-braket/{package}-python.git@{args.branch}\n"
```

Comment:
Would this work if someone is raising a PR from their personal fork. Is there a way we could handle that?

Reply:
I am testing this with a forked branch of the repo. The action runner syncs the repo so this should work across forks with the new [commit](https://github.com/amazon-braket/amazon-braket-sdk-python/pull/792/commits/f33ede55b51ae5659451373b604386cfd4ff6373)